docs/search/criteria_reference/shipment_search_criteria.md

```diff
@@ -0,0 +1,24 @@
+---
+description: Shipment Search Criteria
+edition: commerce
+---
+
+# Shipment Search Criteria reference
+
+Shipment Search Criteria are part of [Search Criteria](search_criteria_reference.md) and they are supported only by Shipment Search.
+
+With these criteria you can filter shipments by their shipment identifier, shipment creation date, shipment status, shipping method, and so on.
+
+## Shipment search criteria
+
+|Search Criterion|Search based on|
+|-----|-----|
+|[CreatedAtCriterion](shipment_createdat_criterion.md)|Date and time when shipment was created|
```

Comment:
I'd cut "Criterion" from all of those, because they are not part of the class name (other than in case of orders).

Reply:
I thought I already fixed that yesterday, let me push my local changes.